src/swift/Sources/Models/TokenFile.swift

```diff
@@ -49,13 +51,15 @@ class TokenFile: Codable {
         case name = "Name"
         case tokens = "Tokens"
         case navigation = "Navigation"
+        case indentLevel = "indentLevel"
```

Comment:
`indentLevel` should not be part of the coding keys and should not be serialized because it is an internal concept only.